crates/solidity/testing/snapshots/cst_output/Expression/incomplete_operand/generated/0.4.11-failure.yml

```diff
@@ -6,15 +6,19 @@ Source: >
 Errors: # 1 total
   - >
     Error: Expected AddressKeyword or BoolKeyword or ByteKeyword or FixedBytesType or FunctionKeyword or Identifier or MappingKeyword or PayableKeyword or SignedFixedType or SignedIntegerType or StringKeyword or UnsignedFixedType or UnsignedIntegerType.
-       ╭─[crates/solidity/testing/snapshots/cst_output/Expression/incomplete_operand/input.sol:1:2]
+       ╭─[crates/solidity/testing/snapshots/cst_output/Expression/incomplete_operand/input.sol:1:9]
        │
      1 │ 2 * new
-       │  ───┬───  
-       │     ╰───── Error occurred here.
+       │         │ 
+       │         ╰─ Error occurred here.
     ───╯
 
 Tree:
   - Expression (Rule): # 0..8 "2 * new\n"
       - NumericExpression (Rule): # 0..1 "2"
           - DecimalLiteral (Token): "2" # 0..1
-      - SKIPPED (Token): " * new\n" # 1..8
+      - BinaryExpression (Rule): # 1..3 " *"
```

Comment:
not sure why `BinaryExpression` is produced with one child (the binary operator) without any operands. Is this expected?